### DISCUSSION TITLE
Larger K8s assets timeout issues

### DISCUSSION THREAD

**Discussion by jake-luna-sada** [mondoohq/cnquery#2603] [General]
Noticing for larger clusters in GCP with over 500 assets im getting timeout errors when trying to sync to console.mondoo,

`error="failed to do request: Post \"https://us.api.mondoo.com/PolicyResolver/SynchronizeAssets\": net/http: request canceled (Client.Timeout exceeded while awaiting headers)"`

Running with --incognito works though it can take some time combing through all assets which is understandable.

**Comment by czunker** [comment]
Thanks  @Jluna-sada for bringing this up.

We will figure out a way to fix this and release a patch.

**Comment by czunker** [comment]
We will track this issue here: https://github.com/mondoohq/cnspec/issues/956

**Comment by jake-luna-sada** [comment]
Awesome thank you, if there is a better avenue for my to submit request just lmk!

**Comment by chris-rock** [comment]
We released many performance updates for cnspec to cover larger accounts. If you still see issues, please feel free to add the issue to this discussion.
